lib/msf/core/handler/bind_aws_instance_connect.rb

```diff
@@ -0,0 +1,347 @@
+# -*- coding: binary -*-
+module Msf
+module Handler
+
+require 'aws-sdk-ec2instanceconnect'
```

Comment:
This require is triggering issues in some rspec, is there a missing gem or some other reason it might not be found?

Maybe this gem [aws-sdk-ec2instanceconnect](https://rubygems.org/gems/aws-sdk-ec2instanceconnect).

I can open a PR that adds that is shortly.